docs/sources/v0.48.x/using-k6/metrics/reference.md

```diff
@@ -33,19 +33,19 @@ These metrics are generated only when the test makes HTTP requests.
+| http_req_duration        | Trend   | Total time for the request. It's equal to `http_req_sending + http_req_waiting + http_req_receiving` (i.e. how long did the remote server take to process the request and respond, without the initial DNS lookup/connection times). `float` |
+| http_req_failed          | Rate    | The rate of failed requests according to [setResponseCallback](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/set-response-callback).                                                                                       |
+| http_req_receiving       | Trend   | Time spent receiving response data from the remote host. `float`                                                                                                                                                                             |
 | http_req_sending         | Trend   | Time spent sending data to the remote host. `float`                                                                                                                                                                                          |
+| http_req_tls_handshaking | Trend   | Time spent handshaking TLS session with remote host                                                                                                                                                                                          |
 | http_req_waiting         | Trend   | Time spent waiting for response from remote host (a.k.a. “time to first byte”, or “TTFB”). `float`                                                                                                                                           |
-| http_req_receiving       | Trend   | Time spent receiving response data from the remote host. `float`                                                                                                                                                                             |
-| http_req_duration        | Trend   | Total time for the request. It's equal to `http_req_sending + http_req_waiting + http_req_receiving` (i.e. how long did the remote server take to process the request and respond, without the initial DNS lookup/connection times). `float` |
-| http_req_failed          | Rate    | The rate of failed requests according to [setResponseCallback](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/set-response-callback).                                                                                                       |
+| http_reqs                | Counter | How many total HTTP requests k6 generated.                                                                                                                                                                                                   |
```

Comment:
`http_req_failed` has been deleted and is unavailable now. Please, restore it.

Reply:
`http_req_failed` is still listed on line 43.

Reply:
apologies

Reply:
no worries! 🙏